LongWarOfTheChosen/Config/XComLW_FactionBalance.ini

```diff
@@ -86,11 +89,19 @@ OVERRIDE_REDUCTION=10
 
 [LW_FactionBalance.X2LWModTemplate_ReaperAbilities]
 SHADOW_DURATION=1
-SHADOW_DETECTION_RANGE_REDUCTION=0.7f
+SHADOW_DETECTION_RANGE_REDUCTION=0.9f

```

Comment:
at this point why not go for 100%?

Reply:
At first, it seemed too cheesy. But then so is 0.9. Now I'm wondering whether it might break things as an edge case. We could try 1.0 and hope for the best. It might also be good to show concealment tiles as an additional indicator that the Reaper is in Shadow.

Reply:
I think the big yellow tint is indication enough, I'd go for 100%.

Reply:
OK, I confirmed that the HUD is still there, so yes, concealment tiles aren't needed to indicate Shadow is active. OK, I'll try 1.0f.